src/executor/contract.rs

```diff
@@ -346,8 +346,7 @@ impl AotContractExecutor {
 
         // There is always a return ptr because contracts always return more than 1 thing (builtin counters, syscall, enum)
         let return_ptr = arena.alloc_layout(unsafe {
-            // 64 = size of enum + builtin sizes
-            // align is 16 because of the u128 from gas
+            // 56 = size of enum
```

Comment:
What does this 56 refer to?

Reply:
the size of the enum in bytes, althoug im using a higher size just in case, but maybe its not needed